com.unity.ml-agents.extensions/Runtime/Sensors/PoseExtractor.cs

```diff
@@ -77,60 +145,80 @@ public int GetParentIndex(int index)
             return m_ParentIndices[index];
         }
 
+        /// <summary>
+        /// Set whether the pose at the given index is enabled or disabled for observations.
+        /// </summary>
+        /// <param name="index"></param>
+        /// <param name="val"></param>
+        public void SetPoseEnabled(int index, bool val)
```

Comment:
SetPoseEnabled is only used to make sure the Virtual Root is not being observed right? I did not see being used elsewhere. Making it public might be dangerous, dot sure what would happen if the user called SetPoseEnabled in the middle of training. 
Also, nothing to do with Enabled / Disabled GameObjects right?

Reply:
At the moment, it's only used on the root (virtual or otherwise). But I was to add a custom Editor to expose it per-body. For example, on the Worm scene, the eyes are actually rigid bodies constrained to the head, with locked linear motion and free angular motion:
![image](https://user-images.githubusercontent.com/6877802/88984771-5c575f00-d283-11ea-8e0e-90793fc9c2a7.png)
We would basically be wasting observations on those, so I'd like to be able to filter them out.

For now, I'd prefer to keep things more accessible, and tighten up the visibility if we move to the core package.

Correct, nothing to do Enabled / Disabled GameObjects. I hadn't thought about that - is there a better name that would make this more obvious?